process/posts_editorial_process.md

```diff
@@ -29,6 +29,6 @@ We'll follow the progress of a post from the idea to publication with [this tabl
 
 ## Process compliance failure
 
-### ETA update
+### Publication date update
 
-If the ETA is modified (this can be a delay or adding a heard deadline), the group responsible for the ETA update (this can be the author during stages 1 to 4 and 8 to 9, or the documentation team during stages 5 to 7) has to tag the other group and the DevRel team in the ETA update PR and make sure that they are aware of this change.
+If the publication date is modified (this can be a delay or adding a heard deadline), the group responsible for the publication date update (this can be the author during stages 1 to 4 and 8 to 9, or the documentation team during stages 5 to 7) has to notify the other group and the DevRel team and make sure that they are aware of this change.
```

Comment:
This was discussed last week with @dichotommy. The documentation team meets twice a week, and they check the post forecast during all of these meetings. So actually notifying them is not super needed unless they do not miss (which could happen if most of them were not working during a specific week).